src/app/register/register-user-form/register-user-form.component.ts

```diff
@@ -1,10 +1,44 @@
+import { MatSnackBar } from '@angular/material/snack-bar';
+import { User } from '../../domain/user';
+import { NewPasswordAndConfirmComponent } from '../../password/new-password-and-confirm/new-password-and-confirm.component';
+import { FormBuilder, FormGroup } from '@angular/forms';
+
 export class RegisterUserFormComponent {
-  NAME_REGEX = '^[a-zA-Z]+([ -]?[a-zA-Z]+)*$';
+  protected NAME_REGEX = '^[a-zA-Z]+([ -]?[a-zA-Z]+)*$';
+
+  protected confirmPasswordLabel: string = $localize`Confirm Password`;
+  protected passwordsFormGroup: FormGroup = this.fb.group({});
+  protected passwordLabel: string = $localize`Password`;
```

Comment:
These variables should be in alphabetical order.

Reply:
done.